neo/rawio/intanrawio.py

```diff
@@ -808,8 +897,11 @@ def read_rhd(filename, file_format: str):
     return global_info, ordered_channels, data_dtype, header_size, BLOCK_SIZE, channel_number_dict
 
 
-###########################
-# RHD Zone for Binary Files
+##########################################################################
+# RHX Zone for Binary Files
```

Comment:
I don't understand this comment here. The verb gives in 902 seems to be missing something.

Reply:
Yeah it's not super clear. I was trying to split the difference between the word "zone" above from the original code and updating it with new wording. What about:

This section of code lists the possible file names in the formats "one-file-per-signal" and "one-file-per-channel" which are optional formats within the rhx intan recording software. They are organized as a series
of headerless binary files (*.dat) with set names.